mix.exs

```diff
@@ -1,7 +1,7 @@
 defmodule ElixirTools.MixProject do
   use Mix.Project
 
-  @version "0.7.0"
+  @version "0.8.0"
```

Comment:
we should not bump the version in the same PR as the functionality

Reply:
if PR is not long lasting and the only PR at the time it should be fine, no?